test/org/sagebionetworks/bridge/dynamodb/DynamoStudyTest.java

```diff
@@ -85,4 +87,46 @@ public void studyFullySerializesForCaching() throws Exception {
         final Study deserStudy = BridgeObjectMapper.get().readValue(json, Study.class);
         assertEquals(study, deserStudy);
     }
+    
+    // Following tests are to check that the minSupportedVersion is always less than or equal to the current version
+    
+    @Test
+    public void testMinSupportedVersionAlwaysLessThanOrEqualToVersion_SetMinLessThan() throws Exception {
+        final DynamoStudy study = TestUtils.getValidStudy(DynamoStudyTest.class);
+        
+        study.setVersion(3L);
```

Comment:
This version value is used by dynamo db to implement optimistic locking, you can take it out of this test. It only confuses what you're testing here.